config.json

```diff
@@ -34,7 +34,7 @@
       "description": "Specifies the distro rid for Unix OS.",
       "valueType": "property",
       "values": [],
-      "defaultValue": "${OSRid}"
+      "defaultValue": "${OSRid}-${CPUArch}"
```

Comment:
When the portable build is run, DistroRid is getting set to rhel.7-x64 (see https://ci.dot.net/job/dotnet_core-setup/job/master/job/GenPRTest/job/release_portablelinux_x64_prtest/1/consoleText) but the packages are being built for the linux-x64 RID (see the output from the packaging task later).

I don't know how in run to say if portable is set, override DistroRid based on that, unfortunately.  I can try to hack that into build.sh but I'm not sure how clean it will be.

Do you know if this is going to be an issue?